Fable.Remoting.Client/Proxy.fs

```diff
@@ -257,7 +264,11 @@ module Proxy =
             /// Sets an authorization string to send with the request onto the Authorization header.
             [<CustomOperation("with_token")>]
             member __.WithToken(state,token) =
-                {state with Headers = (Authorization token)::state.Headers}
+                {state with TokenCallback = (fun _ -> Promise.lift token)}
+            /// Sets a callback which is invoked on every request to acquire a authorization string which will be set onto the Authorization header.
+            [<CustomOperation("with_token_callback")>]
+            member __.WithTokenCallback(state,callback) =
+                {state with TokenCallback = callback}              
```

Comment:
Do I also need to add something to the `createSecureWithEndpointAndBuilderImpl` ?